test/integration-tests/testset_sdk/javascript/testSetSDK.js

```diff
@@ -31,12 +33,76 @@ class TestSetSDK extends TestSetTalent {
         this.registerTest('echoMixedList', [1, 'Hello World', 3.21], this.test_echoMixedList.bind(this), 5000);
         this.registerTest('echoDeepList', [1, [2, [3, [4, [5]]]]], this.test_echoDeepList.bind(this), 5000);
 
+        this.registerTest('receiveEvent1', true, this.test_receiveEvent1ByMultipleTalents.bind(this), 5000);
```

Comment:
Please change the name also for the external "test/request"-name because this is name is visible in the test output (the internal one is not very important but I think it is a good idea if it is aligned)